packages/ethereum-contracts/dev-scripts/run-deploy-contracts-and-token.js

```diff
@@ -1,10 +1,28 @@
 const fs = require("fs");
+const {ethers} = require("hardhat");
+const superTokenFactoryArtifact = require("@superfluid-finance/ethereum-contracts/build/hardhat/contracts/superfluid/SuperTokenFactory.sol/SuperTokenFactory.json");
+const superTokenArtifact = require("@superfluid-finance/ethereum-contracts/build/hardhat/contracts/superfluid/SuperToken.sol/SuperToken.json");
 const {deployContractsAndToken} = require("./deploy-contracts-and-token");
 
 deployContractsAndToken()
     .then(async ({deployer, tokenDeploymentOutput}) => {
         const frameworkAddresses = await deployer.getFramework();
```

Comment:
should check if frameworkAddresses having those addresses you need.

Otherwise it smells that these NFT contracts change how the code should be structured.

Reply:
added the missing addresses to frameworkAddresses to remove these

Reply:
excellent !